### ISSUE TITLE
Intellij plugin 2017.04.03.2 doesn't work with java_proto_library

### ISSUE THREAD

**Issue by dthong** [bazelbuild/intellij#79]
I have a simple BUILD file:
```
filegroup(
    name = "protos",
    srcs = [
        "runway_common.proto",
    ],
)
java_proto_library(
    name = "java",
    imports = [
        "external/com_github_google_protobuf/src/",
    ],
    inputs = [
        "@com_github_google_protobuf//:well_known_protos",
    ],
    protos = [":protos"],
)

```
The error message was:
Error:(10, 1) in @bazel_tools//tools/ide:intellij_info.bzl%intellij_info_aspect aspect on proto_language_deps rule //tenpoint7_proto/runway:java_compile_deps: 

This didn't happen with the previous version.

**Comment by brendandouglas** [comment]
Hi! java_proto_library support isn't present in the latest released version of Bazel (0.4.5), but I also don't expect errors in the aspect.

Which version of Bazel are you using? (and are there additional lines in that error message?)

Could you please include the raw output in the Bazel Console window around that error message?

**Comment by dthong** [comment]
Hi Brendan! Thanks for looking. I was using Bazel 0.4.4. The problem went away when I upgraded bazel to 0.4.5 and Intellij plugin to 2017.04.17.6. So it is good now.
